cfme/infrastructure/cluster.py

```diff
@@ -104,8 +87,21 @@ def is_displayed(self):
             self.entities.title.text == 'Clusters')
 
     toolbar = View.nested(ClusterToolbar)
-    entities = View.nested(ClusterEntities)
-    paginator = PaginationPane()
+    including_entities = View.include(BaseEntitiesView, use_parent=True)
+
+
+class ClusterAllFromProviderView(ClusterView):
```

Comment:
**Optional**
Consider adding a navigation destination to match, even though it has to go through provider details to get there.

Reply:
okay, I'll add new nav destination

Reply:
Thanks @izapolsk, will keep things consistent with other modules that have 'AllForProvider' destinations.

Reply:
@mshriver, creating and using such view will be problematic due to collections. Collections know nothing about provider. So, I moved this view and new nav destination to infra provider.